db/seeds/03_rooms.seeds.rb

```diff
@@ -1,8 +1,11 @@
 # frozen_string_literal: true
 
-puts 'Create rooms'
+puts '~> Create rooms'
 
-4.times do
-  room = FactoryBot.create(:room)
-  room.save
+users = User.with_role_user
+users.each do |user|
+  3.times do
+    room = FactoryBot.create(:room, user:, name: Faker::Company.name)
```

Comment:
FactoryBot.create => mean this record already save to database, so why you add room.save ???